src/sonic-yang-models/yang-models/sonic-configurable-drop-monitor.yang

```diff
@@ -0,0 +1,51 @@
+module sonic-configurable-drop-monitor {
```

Comment:
What's the reason to choose `sonic-configurable-drop-monitor` instead of `sonic-debug-drop-monitor`?